src/main/scala/chargepoint/docile/dsl/CoreOps.scala

```diff
@@ -45,12 +45,14 @@ trait CoreOps extends StrictLogging {
     }
   }
 
+  def awaitIncoming(num: Int)(implicit awaitTimeout: AwaitTimeout): Seq[IncomingMessage] = {
```

Comment:
How do you set the awaitTimeout? I'm thinking it would be good to have a command-line flag to set a  default. Or did you already think of another mechanism?